deploy/tower-operator.yaml

```diff
@@ -5,6 +5,12 @@ metadata:
   creationTimestamp: null
   name: tower-operator
 rules:
+  - apiGroups:
```

Comment:
Same question as above for non-OpenShift.

Reply:
I ran this on a 1.17 KinD cluster and it doesn't appear to cause any issues.